code/public/index.html

```diff
@@ -1,9 +1,9 @@
 <!DOCTYPE html>
 <html lang="en">
-
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" src="./src/index.css" />
```

Comment:
This is unnecessary (and won't actually work when the project is deployed) since the css is being imported in `src/index.js` - it magically becomes part of the build which netlify deploys.